src/main/webapp/app/entities/statistics/statistics.component.html

```diff
@@ -11,8 +11,8 @@ <h2><span class="fas fa-chart-area"></span>&nbsp;<span>Statistics</span></h2>
     <div class="row">
         <div class="col-md-6">
             <h4><label for="reviewFullScreen" class="form-control-label">Report :</label></h4>
-            <div class="row">
-              <select id="reviewFullScreen" class="form-control col-md-6 offset-1" [ngModel]="timeScale" (ngModelChange)="onSelectTimeScale($event)">
+            <span style="display: flex;">
+              <select id="reviewFullScreen" class="form-control col-md-6" [ngmodel]="timeScale" (ngmodelchange)="onSelectTimeScale($event)">
```

Comment:
@ankitbeniwal : There's a build failure; most probably at this line due to `[ngmodel]`. I think it should be `[ngModel]`. Same goes for `[ngmodelchange]`. 

https://github.com/jhipster/jhipster-online/runs/1325032641